database/data-generator.js

```diff
@@ -10,7 +10,7 @@ const NEIGHBOURHOODS = ['Seacliff', 'Haight Ashbury', 'Outer Mission', 'Downtown
 
 const ROOMTYPE = ['Private room', 'Entire home/apt', 'Shared room'];
 
-const STARTDATE = '2018-09-01T23:59:59Z';
+const STARTDATE = '2017-09-01T23:59:59Z';
```

Comment:
I thought we had decided on a start date of 2017-10-01?

Reply:
Yup. I did 12 - 3 = 9 and started there. We are starting at 10-01. Lets just pretend those points are my pre-analysis for now :D. I've updated my script so future generations start at 10-01.